docs/sdk/features/initialization.md

```diff
@@ -23,7 +23,7 @@ If a variable is first read from the cache and has a listener for [realtime upda
 
 ### [• Javascript SDK](/sdk/client-side-sdks/javascript/javascript-install)
 
-### [• React SDK](/sdk/client-side-sdks/react#installation)
+### [• React SDK](/sdk/client-side-sdks/react/react-install)
```

Comment:
we might want to change this to link to Getting Started, since that's where we have useIsDVCInitialized etc